### PR TITLE
Atari add get_kv_diff endpoint

### PR DESCRIPTION
**Pull request by almogdepaz**

### CODE REVIEW

File: tests/core/data_layer/test_data_rpc.py [hunk 1]
```diff
@@ -421,3 +422,91 @@ async def test_get_root_history(one_wallet_node_and_rpc: nodes) -> None:
...
+            {
+                "id": store_id1.hex(),
+                "hash_1": history["root_history"][1]["root_hash"].hex(),
+                "hash_2": history["root_history"][2]["root_hash"].hex(),
+            }
+        )
+        assert len(diff_res["diff"]) == 2
+        diff4 = {"type": "INSERT", "key": key4.hex(), "value": value4.hex()}
+        diff5 = {"type": "INSERT", "key": key5.hex(), "value": value5.hex()}
+        assert diff4 in diff_res["diff"]
```
Comment (fchirica) [inline tests/core/data_layer/test_data_rpc.py:511]: probably not super important, but I would add a {type: "DELETE"} test here as well.

**Review by fchirica — DISMISSED** [review]

File: chia/rpc/data_layer_rpc_api.py [hunk 2]
```diff
@@ -63,7 +63,8 @@ def get_routes(self) -> Dict[str, Callable[[Any], Any]]:
             "/delete_key": self.delete_key,
             "/insert": self.insert,
             "/subscribe": self.subscribe,
-            "/unsubscribe": self.unsubscribe,
+            "/get_kv_diff": self.get_kv_diff,
+            "/get_root_history": self.get_root_history,
```
Comment (altendky) [inline chia/rpc/data_layer_rpc_api.py:67]: Should this be removing the `/unsubscribe` endpoint?  I see that `/get_root_history` isn't being added here but the missing endpoint entry is getting filled in.

```suggestion
            "/unsubscribe": self.unsubscribe,
            "/get_kv_diff": self.get_kv_diff,
            "/get_root_history": self.get_root_history,
```

**Review by altendky — CHANGES_REQUESTED** [review]

File: chia/rpc/data_layer_rpc_api.py (hunk 2 above)
Comment (almogdepaz) [reply chia/rpc/data_layer_rpc_api.py:67]: Thanks for catching, fixed

**Review by altendky — APPROVED** [review]
Approving based on @fchirica's previous approval and the correction to the issue I noticed.

**Pull request merged**